hist/hist/test/test_TH2Poly_GetNumberOfBins.cxx

```diff
@@ -0,0 +1,14 @@
+// test TH2Poly GetNumberOfBins
+
+#include "gtest/gtest.h"
+
+#include "TH2Poly.h"
+
+TEST(TH2Poly, GetNumberOfBins)
+{
+   TH2Poly *h2p = new TH2Poly();
+   EXPECT_EQ(0, h2p->GetNumberOfBins());
+
+   h2p->AddBin(1, 1, 1, 1);
+   EXPECT_EQ(1, h2p->GetNumberOfBins());
```

Comment:
Would it be possible to do it on the stack? The histogram will otherwise cause a memory leak.

```suggestion
   TH2Poly h2p;
   EXPECT_EQ(0, h2p.GetNumberOfBins());

   h2p.AddBin(1, 1, 1, 1);
   EXPECT_EQ(1, h2p.GetNumberOfBins());
```